src/any_agent/tools/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .a2a_tool import a2a_query
```

Comment:
Unless we make the imports lazy in the `a2a_tool`, this will cause any tool import to crash if `a2a` is not installed